### ISSUE TITLE
Hi, are people still using this?

### ISSUE THREAD

**Issue by artisavotins** [artisavotins/react-redux-i18n#55]

**Comment by pdlanl** [comment]
@artisavotins I have been using it for most of my projects

**Comment by marcosalcazar** [comment]
@artisavotins me too!

**Comment by corneliushka** [comment]
Hi, I'm still using this too since it works well with Redux and is simple to implement. 
Updates are welcome :)

**Comment by dissiCementys** [comment]
Hello, I am using it now. Updates are welcome. Thanks

**Comment by ramprakashram** [comment]
I am using it too, it would be better if you update it. Thanks.

**Comment by edcolcode** [comment]
Hi, I am using it too. Thanks for your amazing job. Updates are welcome. Thanks again!

**Comment by ramprakashram** [comment]
@artisavotins I have made a PR (#50 ) upgrading to latest version of react-i18nify (3.2.5), which supports new features like replacements.
Please review and accept it soon.

**Comment by mahmood-saghrajooghi** [comment]
yes. I'm currently using it. and i have some problems switching to redux toolkit

**Comment by gencurrent** [comment]
I plan to give it a try.
Your update would be kindly appreciated

**Comment by jarnix** [comment]
of course we are using it :)

**Comment by ArthurHVS** [comment]
Yep, in production! Cheers.
